packages/libs/react-ui/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "declarationDir": "types",
     "outDir": "dist/esm",
     "allowJs": true,
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```

Comment:
What was the reason for this change?

Reply:
`node` resolution doesn't support modern `exports` fields in `package.json` and react-ui uses those from crackle, so I was getting errors can't resolve `@kendena/react-ui/styles`, so I had to switch to a modern module resolution like `node16` or `bundle` and since bundle is the recommend way with bundlers I used it, in all the apps and react-ui as well